README.md

```diff
@@ -1,133 +1,126 @@
 Chitter Challenge
 =================
 
-* Challenge time: rest of the day and weekend, until Monday 9am
-* Feel free to use Google, your notes, books, etc. but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday morning
+([Link to original README](./original-README.md))
 
-Challenge:
--------
+## Getting started
 
-As usual please start by forking this repo.
+_PostgreSQL should be installed_
 
-We are going to write a small Twitter clone that will allow the users to post messages to a public stream.
+```sh
+git clone git@github.com:dafuloth/chitter-challenge.git
```

Comment:
The link does not work, managed to clone using this one: git clone https://github.com/dafuloth/chitter-challenge.git

Reply:
Glad at least one way worked for you :) I think for some both will work, or just one of them